ui/src/FindBar.tsx

```diff
@@ -1,23 +1,53 @@
 import React from "react";
+import { convertToAnnotationId } from './selectors/annotation'
 import { EventBus } from "./types/pdfjs-viewer";
+import { ScholarReaderContext } from "./state";
+import { BoundingBox } from "./types/api";
+
+interface FindBarProps {
+  mappingToBounds: Map<String, BoundingBox>;
+  matches: string[];
+  selectedSymbol: string;
```

Comment:
Might it make more sense to access these three properties through `ScholarReaderContext.Consumer`? I think that's the pattern that tends to be used across the other components to get access to information about global state of what's selected.

Reply:
(I also see your comment below in ScholarReader about something needing to be a prop. I'm not sure I understand for now---let's catch up over a video call soon.)

Reply:
Ah I'm not sure I quite understand, what would that look like accessing it through context? I don't believe it would be possible to access it via the render method since we need to update the state.